Slam/DefaultAddMatchInteractor.swift

```diff
@@ -2,12 +2,20 @@ import Foundation
 
 public class DefaultAddMatchInteractor : AddMatchInteractor {
 
+    var api : SlamAPI?
     let view : AddMatchView
+
     public init(view: AddMatchView) {
         self.view = view
     }
+    
+    public init(view: AddMatchView, api: SlamAPI) {
+        self.view = view
+        self.api = api
+    }
 
     public func attemptAdd(#playerOne: String, playerTwo: String) {
+        api!.addMatch(playerOne, playerTwo: playerTwo)
```

Comment:
When using optionals in production code, favor using `?` over `!`. It's safer.

In the tests, the opposite rule applies, since you want to fail fast.